inc/saiport.h

```diff
@@ -1698,6 +1749,82 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_SYSTEM_PORT,
 
+    /**
+     * @brief MDIX status to the user
+     *
+     * @type sai_port_mdix_mode_status_t
+     * @flags READ_ONLY
+     */
+    SAI_PORT_ATTR_MDIX_MODE_STATUS,
+
+    /**
+     * @brief Configure MDIX mode
+     *
+     * @type sai_port_mdix_mode_t
+     * @flags CREATE_AND_SET
+     * @default SAI_PORT_MDIX_MODE_AUTO
+     */
+    SAI_PORT_ATTR_MDIX_MODE,
+
+    /**
+     * @brief Configure Auto Neg mode
+     *
+     * @type sai_port_auto_neg_config_mode_t
+     * @flags CREATE_AND_SET
+     * @default SAI_PORT_AUTO_NEG_CONFIG_MODE_DISABLED
+     */
+    SAI_PORT_ATTR_AUTO_NEG_CONFIG_MODE,
+
+    /**
+     * @brief Enabled auto detection between 1000X and SGMII slave mode
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_1000X_SGMII_SLAVE_AUTODETECT,
+
+    /**
+     * @brief EEE latency mode. Fixed latency - False, Variable latency - True
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_EEE_LATENCY_MODE,
+
+    /**
+     * @brief Enable Auto medium to select both Fiber and Copper media types
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_AUTO_MEDIUM,
+
+    /**
+     * @brief Configure media precedence.
+     *
+     * The PHY supporting mutually exclusive copper and fiber ports should be
+     * configured to select the preferred/active media.
+     * True will configure Fiber ports to be active and False will configure
```

Comment:
* True:  Configure Fiber ports to be active 
* False Configure Copper ports to be active